content/docs/studio/user-guide/projects-and-experiments/visualize-and-compare.md

```diff
@@ -73,6 +72,15 @@ experiment, you can use the plots to compare them.
 
 ![](https://static.iterative.ai/img/studio/plots.png)
 
+### Live (real-time) plots
```

Comment:
```suggestion
### Live plots (real time)
```

Reply:
Didn't change this. Let me know if you strongly feel about this.

Reply:
If it's the main section about the feature and the feature is called "Live plots" I do think we should word it consistently (for SEO at least).

Reply:
It's not the main section about the feature, as there is a separate page for Live metrics and plots. But it's ok to remove `real-time` from the title. I've changed it.